wls-vorfaelleundvorkommnisse-service/src/main/java/de/muenchen/oss/wahllokalsystem/vorfaelleundvorkommnisseservice/rest/ereignis/EreignisController.java

```diff
@@ -20,69 +20,67 @@
 import org.springframework.web.bind.annotation.ResponseStatus;
 import org.springframework.web.bind.annotation.RestController;
 
-/**
- * This controller is responsible for mapping businessActions to REST-endpoints.
- */
 @RestController
-@AllArgsConstructor
-@RequestMapping(value = "/businessActions")
+@RequiredArgsConstructor
+@RequestMapping(value = "/businessActions/ereignisse")
 public class EreignisController {
 
-    EreignisDTOMapper ereignisDTOMapper;
+    private final EreignisDTOMapper ereignisDTOMapper;
 
-    EreignisService ereignisService;
+    private final EreignisService ereignisService;
 
-    /**
-     * This BusinessAction's purpose is: Laden der besonderen Ereignisse
-     * It returns one Ereignis.
-     */
     @Operation(description = "Laden der Ereignisse des Wahllokals {wahlbezirkID}")
     @ApiResponses(
             value = {
                     @ApiResponse(
                             responseCode = "200", description = "OK",
                             content = { @Content(mediaType = "application/json", schema = @Schema(implementation = WahlbezirkEreignisseDTO.class)) }
                     ),
-                    @ApiResponse(
-                            responseCode = "500", description = "Probleme bei der Kommunikation mit dem externen System von dem die Daten importiert werden",
-                            content = { @Content(mediaType = "application/json", schema = @Schema(implementation = WlsExceptionDTO.class)) }
-                    ),
                     @ApiResponse(
                             responseCode = "204", description = "Keine Daten vom Fremdsystem geliefert",
                             content = @Content(schema = @Schema())
+                    ),
+                    @ApiResponse(
+                            responseCode = "400", description = "Anfrageparameter sind fehlerhaft",
+                            content = @Content(mediaType = "application/json", schema = @Schema(implementation = WlsExceptionDTO.class))
+                    ),
+                    @ApiResponse(
+                            responseCode = "500", description = "Probleme bei der Kommunikation mit dem externen System von dem die Daten importiert werden",
+                            content = { @Content(mediaType = "application/json", schema = @Schema(implementation = WlsExceptionDTO.class)) }
                     )
```

Comment:
Der Service hat keine Abhängigkeit zu anderen Services. Ein 500er kann daher nur bei sonstigen internen Problemen auftreten.

Reply:
siehe commit [992d6ba](https://github.com/it-at-m/Wahllokalsystem/pull/412/commits/992d6ba909338cab856f1c85081656e3844bbaa8)